.github/CODEOWNERS

```diff
@@ -131,8 +131,8 @@
 #/avm/res/sql/managed-instance/ @Azure/avm-res-sql-managedinstance-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/sql/server/ @Azure/avm-res-sql-server-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/storage/storage-account/ @Azure/avm-res-storage-storageaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/synapse/private-link-hub/ @Azure/avm-res-synapse-privatelinkhub-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/synapse/workspace/ @Azure/avm-res-synapse-workspace-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/synapse/private-link-hub/ @Azure/avm-res-synapse-privatelinkhub-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Isn't it now out of alphabatical order? Or do the tester tests fail at the original location?

Reply:
Oh sure! i've missed it! Updated now. on my fork all was green

Reply:
Looks still like the order changed. Could you move line 135 back to 134 and check if the static tests fail? Because if that's the case there may be a bug in the static tests we need to fix

Reply:
Ok, this was my bad. The CODEOWNERS file does not need an order. Resolving